api/v1alpha1/component_types.go

```diff
@@ -38,18 +39,11 @@ type GitSource struct {
 	DockerfileURL string `json:"dockerfileUrl,omitempty"`
```

Comment:
Is there an order of precedence for component creation if the URL, DevfileURL, and DockerfileURLs are all specified at once?  Should this be documented?

Reply:
Right now, they are placeholders.  Those are currently not used by the Generator - only the URL is used.

Reply:
Based on your later review comments, I'll get rid of these.

Reply:
I don't know if this will break HAS.  If these are HAS specific, then they belong in the HAS repo.  They don't really fit the category of "git source" either. cc @maysunfaisal